express/parsers/apps/espresso/parser.py

```diff
@@ -22,7 +22,20 @@ def __init__(self, *args, **kwargs):
         self.work_dir = self.kwargs["work_dir"]
         self.stdout_file = self.kwargs["stdout_file"]
         self.txt_parser = EspressoTXTParser(self.work_dir)
-        self.xml_parser = EspressoXMLParser(find_file(settings.XML_DATA_FILE, self.work_dir))
+        self.xml_parser = EspressoXMLParser(self.find_xml_file())
+
+    def find_xml_file(self):
+        """
+        Finds XML file. GW XML file is returned if this a GW calculation.
```

Comment:
We should be careful with naming here b/c QE has multiple GW implementations (eg. GWW). We should call GW "SteinheimerGW" everywhere instead